soap/src/java/com/zimbra/soap/mail/message/CreateSearchFolderRequest.java

```diff
@@ -34,13 +36,15 @@
  */
 @XmlAccessorType(XmlAccessType.NONE)
 @XmlRootElement(name=MailConstants.E_CREATE_SEARCH_FOLDER_REQUEST)
+@GraphQLType(name=MailConstants.E_CREATE_SEARCH_FOLDER_REQUEST, description="create search folder request")
 public class CreateSearchFolderRequest {
```

Comment:
You don't need to annotate anything in this class, it isn't being exposed. Remove all changes to it.

Reply:
done